doc/whats_new/v1.4.rst

```diff
@@ -152,6 +152,14 @@ Changelog
   works with pandas copy-on-write mode enabled.
   :pr:`28233` by :user:`John Hopfensperger <s-banach>`.
 
+
+:mod:`sklearn.impute`
```

Comment:
You can move this section under inspection. We will reorder the rest properly for the release.